spring-cloud-task-core/src/main/resources/org/springframework/cloud/task/schema-sqlserver.sql

```diff
@@ -0,0 +1,18 @@
+CREATE TABLE TASK_EXECUTION  (
+  TASK_EXECUTION_ID BIGINT  NOT NULL PRIMARY KEY ,
+  START_TIME DATETIME DEFAULT NULL ,
+  END_TIME DATETIME DEFAULT NULL ,
+  TASK_NAME  VARCHAR(100) ,
+  EXIT_CODE INTEGER ,
+  EXIT_MESSAGE VARCHAR(2500) ,
```

Comment:
Given that we store stack traces here...do we want to convert these to something larger (for example `TEXT` on MySql)?